internal/ethapi/api.go

```diff
@@ -814,8 +812,10 @@ func DoEstimateGas(ctx context.Context, b Backend, args CallArgs, blockNr rpc.Bl
 	return hexutil.Uint64(hi), nil
 }
 
+// EstimateGas returns an estimate of the amount of gas needed to execute the
+// given transaction against the current pending block.
 func (s *PublicBlockChainAPI) EstimateGas(ctx context.Context, args CallArgs) (hexutil.Uint64, error) {
-	return DoEstimateGas(ctx, s.b, args, rpc.PendingBlockNumber)
+	return s.doEstimateGas(ctx, s.b, args, rpc.PendingBlockNumber)
```

Comment:
@Arachnid I would like you to confirm this change. My understanding reading your PR was that you wanted to make `DoCall` and `DoEstimate` independent. And in fact, this is what is [currently breaking the build](https://travis-ci.org/ethereum/go-ethereum/jobs/480056662#L808). Could you and @kshinn clarify what you want to do here?

Reply:
@kshinn This breaks existing code (per the build):

```
# github.com/ethereum/go-ethereum/graphql
graphql/grahpql.go:894:30: undefined: ethapi.DoCall
graphql/grahpql.go:915:14: undefined: ethapi.DoEstimateGas
```

Would you mind either reverting the change or fixing that code? I'm okay with either if @gballet is.

Reply:
Don't mind at all. I'll play with getting this working and push and update to this branch.